pom.xml

```diff
@@ -63,7 +63,7 @@
     <gwteventservice.version>1.2.1</gwteventservice.version>
     <okapi.version>0.22</okapi.version>
 
-    <zanata.assets.version>7.4-SNAPSHOT</zanata.assets.version>
+    <zanata.assets.version>8.0-SNAPSHOT</zanata.assets.version>
```

Comment:
@djansen-redhat 

> Translate / Review permissions list very much not in order

Is this:

 - the order of individual permissions listed next to a person (when the sorting is alphabetical)?
 - the order of locale headings when grouped by role?
 - the order of people within a locale when grouped by role?
 - something else?